src/crepe/facade/SDLContext.h

```diff
@@ -8,9 +9,13 @@
 #include <functional>
 #include <memory>
 #include <string>
+#include <utility>
 
+#include "../api/Event.h"
+#include "../api/KeyCodes.h"
 #include "../api/Sprite.h"
 #include "../api/Transform.h"
+#include "../api/Vector2.h"
```

Comment:
Volgens mij is deze include niet nodig. Als het goed is werk je d.m.v. één van de standaard gedefinieerde Vector2 in types.h.